### PR TITLE
Updated Error Text, fixed var typo

### PR DESCRIPTION
**Pull request by JesperDramsch**
Made logs more verbose and informative for user debugging.
Fixed typo in `playlistIdw`.

Figured a separate PR for this is appropriate @dudemario

### CODE REVIEW

File: sheetScript.gs [hunk 1]
```diff
@@ -225,7 +225,7 @@ function getVideoIds(channelId, lastTimestamp) {
       }
     } catch (e) {
       Logger.log("Cannot search YouTube, ERROR: " + e.message);
-      break;
+      return [];
```
Comment (JesperDramsch) [inline sheetScript.gs]: This is consistent with the rest of the script. Runs, but returns an empty list.

Comment (JesperDramsch) [reply sheetScript.gs]: This was concurrent with @dudemario's changes from `return videoIds` to `break`. This change makes sure the log still writes to the sheet.

Comment (dudemario) [reply sheetScript.gs]: Sounds reasonable, similar case for [line 287](https://github.com/Elijas/auto-youtube-subscription-playlist-2/blob/1b3cdef21780cdc6790bd19447b53e8eea465ca5/sheetScript.gs#L287)

Comment (JesperDramsch) [reply sheetScript.gs]: Changed it to `break`. Your solution is much better.

**Comment by dudemario** [comment]
Thanks for catching that typo

**Pull request merged**